src/test/java/hudson/plugins/ec2/EC2PrivateKeyTest.java

```diff
@@ -41,6 +44,12 @@
     @Rule
     public JenkinsRule r = new JenkinsRule();
 
+    @Before
+    public void init(){
+        // Tests using the BouncyCastleProvider failed without that
+        Security.addProvider(new org.bouncycastle.jce.provider.BouncyCastleProvider());
```

Comment:
All the tests fail without this?

Reply:
Probably an issue in Java 9+? I read about an incompatibility with the module path and how JCA is designed, but I can't find an official bug about it.

Reply:
For some reason this is not possible anymore on my current machine. I removed it for now.